src/zodbbrowser/browser.py

```diff
@@ -1,5 +1,6 @@
 import time
 import logging
+import pkg_resources
```

Comment:
pkg_resources is not a standard library module, so it should be moved next to transaction/simplejson imports.